app/models/ems_refresh/save_inventory_container.rb

```diff
@@ -2,8 +2,8 @@ module EmsRefresh::SaveInventoryContainer
   def save_ems_container_inventory(ems, hashes, target = nil)
     target = ems if target.nil?
 
-    child_keys = [:container_projects, :container_nodes, :container_image_registries, :container_images,
-                  :container_replicators, :container_groups, :container_services, :container_routes]
+    child_keys = [:container_projects, :container_quotas, :container_limits, :container_nodes, :container_image_registries,
```

Comment:
I believe this order makes sense: resource_quotas and limit_ranges are both per namespace 
for this reason I've placed them after container_groups.